examples/chip-tool-darwin/commands/clusters/ReportCommandBridge.h

```diff
@@ -44,6 +44,7 @@ class ReadAttribute : public ModelCommand {
         : ModelCommand("read")
     {
         AddArgument("attr-name", attributeName);
+        AddArgument("data-version", 0, UINT32_MAX, &mDataVersion);
```

Comment:
Shouldn't the read-by-id versions also allow this?

Reply:
I think it should also be included in the read-by-id, but the CHIP.framework does not support passing in this.

Reply:
OK.  Can you please make sure an issue is filed?

Reply:
https://github.com/project-chip/connectedhomeip/issues/18334